modules/extensions/src/path-style/shaders.glsl.js

```diff
@@ -57,3 +57,32 @@ float round(float x) {
 `
   }
 };
+
+export const offsetShaders = {
+  inject: {
+    'vs:#decl': `
+attribute float instanceOffsets;
+varying float vOffset;
+`,
+    'vs:DECKGL_FILTER_SIZE': `
+  float offsetScale = abs(instanceOffsets * 2.0) + 1.0;
+  size *= offsetScale;
+  vOffset = instanceOffsets;
+`,
+    'fs:#decl': `
+varying float vOffset;
+`,
+    'fs:#main-start': `
+  float isInside;
+  float offsetScale = (abs(vOffset * 2.0) + 1.0);
+  if (vOffset >= 0.0) {
+    isInside = step(vOffset * 2.0 - 1.0, vPathPosition.x * offsetScale);
+  } else {
+    isInside = step(vPathPosition.x * offsetScale, vOffset * 2.0 + 1.0);
+  }
+  if (isInside == 0.0) {
```

Comment:
Nice sculpting :)

Reply:
Eventually I want to use `smoothedge` and apply the result to the alpha. Right now this is not possible because I can only safely modify the color via the `FILTER_COLOR` hook, in which I don't have access to `vPathPositions`.